src/functions/usduser.md

```diff
@@ -1,51 +1,46 @@
+---
+description: Get specific user's information.
+---
+
 # $user
 
-This function returns the given user's specified property
+This function returns the given user's specified property.
+
+### Usage
 
-#### Usage
+```php
+$user[userID;option?]
+```
 
-This function has 2 fields
+> If left blank, returns user's username.
 
-1. User ID \(Required\)
-2. Property \(Required\)
+### Fields
 
-Raw Usage: `$user[userID;property]`
+| Field | Description | Type | Required |
+| :--- | :--- | :--- | :--- |
+| userID | The ID of the user | number | yes |
+| option | The property of we want to get | string | no |
 
 #### Options
 
-* User ID - The user the properties are based off of
-* Property - The property we're getting from &lt;user&gt;
-
-#### Available Properties
+* `accentColor` — Returns custom accent color
+* `avatarURL` — Returns user's avatar URL
+* `banner` — Returns hash of banner
+* `bannerURL` — Returns user's banner URL
+* `createdAt` — Returns account's created time
+* `createdTimestamp` - Returns account's created timestamp
+* `flags` — Returns user's flags
+* `partial` — Returns the user data is available or not as boolean
+* `dmChannel` — Returns dm channel ID between bot and the user.
```

Comment:
there are more

Reply:
What do you mean by more? That's what I saw on `function.js` source code. I've added default option too, I guess that's all.

Reply:
function.js adds getter data / function call data into an object

because assigning a class props to object removes getters and methods

default options like `id` , `username` etc are also present

Reply:
Where can I find id, username, tag, discriminator etc?